src/styles/settings.plugin.scss

```diff
@@ -110,3 +113,17 @@
 .jw-settings-submenu-button {
 	display: none !important;
 }
+
+// caption styling
+// @see https://developer.mozilla.org/en-US/docs/Web/CSS/::cue
+::cue {
+	background-color: rgba(0, 0, 0, 0.6) !important;
```

Comment:
Is that needed to override JW styles?

Reply:
no need for that, see https://github.com/Wikia/jwplayer-fandom/pull/19/files#diff-328f91fce83d078dfc42789912f79cbbR103